### DISCUSSION TITLE
Can we use StringBuffer class instead of StringBuilder?

### DISCUSSION THREAD

**Discussion by elizabethoswald** [TEAMMATES/teammates#12919] [Ideas]
Hi, In the src/client/java/teammates/client/scripts/ListActiveInstructors.java, can we use StringBuffer class instead of StringBuilder as the StringBuffer class is thread-safe one?

**Comment by slraina** [comment]
Hi @elizabethoswald sorry to post it here since i don't know where to,
Actually this is the first project, i forked and cloned, But now i have absolutely no idea how to proceed further. Please help me!

**Comment by cedricongjh** [comment]
see: https://github.com/TEAMMATES/teammates/issues/13055#issuecomment-2063549855
